index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Registration Form</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/css/styles.css" />
+    <link rel="stylesheet" href="/css/registration-form.css" />
+</head>
+<body>
+    <header>
+        <h1>Registration Form</h1>
+    </header>
+    <main>
+        <h2>Corn Maze Volunteer Registration</h2>
+        <form id="volunteer-registration">
+            <label id="name-prompt">Name:</label>
+            <input type="text" required name="name" id="name-entered">
+            <label id="pronoun-prompt">Preferred Pronoun:</label>
+            <input type="text" required name="pronoun" id="pronoun-entered">
+            <label id="email-prompt">Email:</label>
+            <input type="email" required name="email" id="email-entered">
+            <fieldset>
+                <legend>
+                    <h3>Do a Thing, Get a T-Shirt!</h3>
+                </legend>
+                <h4 id="size">Size</h4>
+                <input type="radio" required id="small" name="size" value="small">
+                <label for="small">Small</label>
+                <input type="radio" required id="medium" name="size" value="medium">
+                <label for="medium">Medium</label>
+                <input type="radio" required id="large" name="size" value="large">
+                <label for="large">Large</label>
+                <input type="radio" required id="x-large" name="size" value="x-large">
+                <label for="x-large">X-Large</label>
+                <div>
+                    <h4 id="style">Style</h4>
+                    <input type="radio" required id="unisex" name="style" value="unisex">
+                    <label for="unisex">Unisex</label>
+                    <input type="radio" required id="fitted" name="style" value="fitted">
+                    <label for="fitted">Fitted</label>
+                    <input type="radio" required id="kids" name="style" value="kids">
+                    <label for="kids">Kids</label>
+                </div>
+                <div>
+                    <br>
+                    <label for="color"><strong>Color:&nbsp</strong></label>
+                    <select id="color" name="color">
```

Comment:
```suggestion
                    <select id="color" name="color">
                       <option value="" disabled selected>Select your option</option>
                       <option value="hurr">Durr</option>
                    </select>
```
You can have a disabled option that allows you to have a title/placeholder for your dropdown. Also, be mindful of your HTML indentation -- you want child elements to be nested properly.